intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -264,16 +222,17 @@ all.equal(stats_nopipe, stats_pipe)
 
 `all.equal()` is letting us know that these two objects are the same. 
 
-Now that hopefully you are convinced that the tidyverse can help you make your 
-code neater and easier to use and read, let's go through some of the popular 
-tidyverse functions and so we can create pipelines like this. 
+Now that hopefully you are convinced that the tidyverse can help you make your code neater and easier to use and read, let's go through some of the popular tidyverse functions and so we can create pipelines like this. 
+
+_Note:_ We are using the `magrittr` package and the pipe that comes with it, `%>%`, throughout this course. 
+After seeing the wide adoption of this pipe, R core team decided this was such a useful feature that they incorporated a version of it in the native R language from version 4.1 up.
+This "native" pipe is `|>`, and can be used just about anywhere you might have used `%>%` (though there are some [subtle differences](https://www.jumpingrivers.com/blog/new-features-r410-pipe-anonymous-functions/#the-native-pipe).
```

Comment:
Add something like - 
> The "native" pipe is composed of two symbols: the vertical bar `|`, (itself also known as "pipe" in different programming languages or contexts!), and the greater than symbol `>`. The vertical bar is _not the letter "L" or "i"_,and it is located above the `Return` (or `Enter`) key on your keyboard.

Reply:
I agree that if we were to be using this, we would want more explanation, but since we are not planning to use it, I might leave this as something that could be said but maybe not written here for now? I mostly just want to keep this as brief as possible.

Reply:
Sounds good to me.